### PR TITLE
Add registry list command

### PR DESCRIPTION
**Pull request by 12345lcr**
Fixed: https://github.com/knative-sandbox/kn-plugin-admin/issues/6

* Add registry list command
* Add service account tag to those registry secrets who stored in this service account's `imagePullSecret`
* Add unit test and integration test for registry list command

### CODE REVIEW

**Comment by 12345lcr** [comment]
@googlebot I signed it!

**Comment by 12345lcr** [comment]
@googlebot I signed it!

File: pkg/command/registry/list.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	"fmt"
+	"sort"
+
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+
+	"knative.dev/client-contrib/plugins/admin/pkg"
```
Comment (zhanggbj) [inline pkg/command/registry/list.go:27]: It's good to wait and rebase after https://github.com/knative-sandbox/kn-plugin-admin/pull/8 is merged (Currently it is under review.), as we need to update package import path. Thanks!

**Comment by zhanggbj** [comment]
@12345lcr Thanks for the PR! Looks good and I just left one comment about the package. Thanks!

**Comment by zhanggbj** [comment]
/approve
/lgtm
Thanks @12345lcr for your PR👍 Merging it.

**Pull request merged**